tests/cases/fourslash/goToDefinitionOverriddenMember14.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="./fourslash.ts"/>
+
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @noImplicitOverride: true
+// @filename: a.js
+
+////class Foo {
+////    /*2*/m() {}
+////}
+////class Bar extends Foo {
+////    /** [|@override method {|"name": "1"|}comment |]*/
+////    m() {}
+////}
+
+verify.goToDefinition("1", "2");

```

Comment:
Emmmmm. That's the point. I thought we should not goto definition when click the comments of jsdoc tag.

Reply:
For example:

```
class C {
  /**
   * @override see {@link http://google.com} balabala.
   */
  foo () {

  }
}
```